ansible/group_vars/nimbus.fluffy.yml

```diff
@@ -23,8 +23,18 @@ nimbus_fluffy_consul_success_before_passing: 30
 nimbus_fluffy_consul_failures_before_warning: 1440
 nimbus_fluffy_consul_failures_before_critical: 2880
 
-# Open Ports
-open_ports_default_comment: 'SMART Metrics'
-open_ports_default_chain: 'VPN'
+# Migrated to NFTables from IPTables.
+# https://github.com/status-im/infra-misc/issues/301
+bootstrap__firewall_nftables: true
+
+# Open Ports -------------------------------------------------------------------
+host_el_type: '{{ ansible_hostname|split("-")|first }}'
 open_ports_list:
-  - { port: '{{ smart_metrics_listen_port }}', ipset: 'metrics.hq' }
+  nimbus-fluffy:
+    - { comment: 'Nimbus Fluffy',          port: '{{ nimbus_fluffy_listening_port }}',                                      protocol: 'udp' }
+    - { comment: 'Nimbus Fluffy Metrics',  port: '9201-9232',                          ipset: 'metrics.hq', iifname: 'wg0', protocol: 'tcp' }
+  nimbus-portal:
+    - { comment: 'Nimbus Portal',          port: '19100',                                                                   protocol: 'udp' }
+    - { comment: 'Nimbus Portal Metrics',  port: '19200',                              ipset: 'metrics.hq', iifname: 'wg0', protocol: 'tcp' }
```

Comment:
Isn't it here?
https://github.com/status-im/infra-nimbus/blob/master/ansible/vars/portal-bridge.yml

Reply:
The problem is that it will clash with this: https://github.com/status-im/infra-nimbus/blob/b4414336aba03b6a3991ee7aa02ab270d4405f76/ansible/group_vars/nimbus.fluffy.yml#L11C1-L11C37

Reply:
I don't understand that configuration.
portal-bridge.yml and fluffy.yml are both for fluffy fleet

Reply:
So I guess we need to keep firewall rules in one place.